src/components/IstioActions/IstioActionsDropdown.tsx

```diff
@@ -56,23 +59,34 @@ class IstioActionDropdown extends React.Component<Props, State> {
     this.props.onDelete();
   };
 
+  renderTooltip = (key, position, msg, child): any => {
```

Comment:
Better to move the "any" types per proper ones, on this case "React.ReactFragment" could be the appropiate I think.
There could be some "any" use in the code but when we can, let's try to move it to strong types, that would help future migrations.